.gitlab-ci.yml

```diff
@@ -34,6 +34,16 @@ stages:
   except:
     - tags
 
+.windows-test-base:
+  stage: test
+  before_script:
+    - virtualenv venv3
+    - venv3\Scripts\activate
+    - pip install -r requirements-dev.txt
+    - .\scripts\fetch_secret.py --secret-name gcp-credentials.json > gcp-credentials.json
+    - $env:GOOGLE_APPLICATION_CREDENTIALS = (Convert-Path .\gcp-credentials.json)
```

Comment:
Can this simply be changed to the following?
```
    - $env:GOOGLE_APPLICATION_CREDENTIALS = ./gcp-credentials.json

```

Reply:
i dont believe so, addin Convert-Path will expand out the path to the directory just like, `export GOOGLE_APPLICATION_CREDENTIALS=$(pwd -P)/gcp-credentials.json` does in the linux base.